service/resource/vaultcrtv2/update_test.go

```diff
@@ -270,7 +283,128 @@ func Test_Resource_VaultCrt_shouldCertBeRenewed(t *testing.T) {
 			}
 		}
 
-		result, err := newResource.shouldCertBeRenewed(tc.Secret, tc.TTL, tc.Threshold)
+		result, err := newResource.shouldCertBeRenewed(tc.Secret, tc.Secret, tc.TTL, tc.Threshold)
```

Comment:
The desired secret does not matter here so we simply reuse the given one because we need a kind of useful one.